docs/en/operations/table_engines/tinylog.md

```diff
@@ -1,22 +1,12 @@
 # TinyLog
 
-Engine belongs to the family of log engines. See the common properties of log engines and their differences in the [Log Engine Family](log_family.md) article.
+Engine belongs to the family of log engines. See [Log Engine Family](log_family.md) for common properties of log engines and for their differences.
 
-The simplest table engine, which stores data on a disk.
-Each column is stored in a separate compressed file.
-When writing, data is appended to the end of files.
+The typical way using this table engine is write-once method: firstly write the data one time, then read it as many times as needed. For example, you can use `TinyLog`-type tables for intermediary data that is processed in small batches.
 
-Concurrent data access is not restricted in any way:
```

Comment:
this was a pretty important warning

Reply:
If I've understood the #6226 pull right, then TinyLog now supports locks for concurret data access along with other engines of the family.